Makefile

```diff
@@ -58,3 +79,8 @@ define go
 	$(ENV_PREFIX) go $(1) -tags "fts5" -ldflags "-X=main.Version=$(VERSION) -X=main.Build=$(BUILD)" $(2)
 endef
 
+# Alpine (musl) requires statically linked libs. This should be compatible for
```

Comment:
not sure if this is the cleanest way to simply add `-extldflags=-static` to the build case for alpine? But couldn't come up with anything else.